code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -354,7 +354,8 @@
 	icon = initial(icon)
 	icon_state = icon_living
 	density = initial(density)
-	flying = initial(flying)
+	if(TRAIT_FLYING in initial_traits) //CHAP-TODO: Unfuck this
```

Comment:
Do you plan on fixing this TODO in this PR, or gonna save it for when you plan on expanding initial_traits? It's fine matter either way, I just want to know what the scope of this is.

Reply:
Remove the comment, was just leftover